docs/howto/kubernetes/gardener/kubectl.md

```diff
@@ -3,40 +3,65 @@ description: How to fetch, verify, and use your kubeconfig with kubectl.
 ---
 # Managing a Kubernetes cluster
 
-Once you [have launched a new cluster](create-shoot-cluster.md), you can interact with it using `kubectl` and a [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file.
+Once you [have launched a new cluster](create-shoot-cluster.md), you can interact with it using `kubectl` with a [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig) file.
 
 ## Prerequisites
 
-You must install the Kubernetes command line tool, `kubectl`, on your local computer, and run commands against your cluster.
-To install `kubectl`, follow [the relevant Kubernetes documentation](https://kubernetes.io/docs/tasks/tools/#kubectl).
+You should install `kubectl`, the Kubernetes command line tool, on your local computer.
+Then, you will be able to issue commands against your clusters.
+To install `kubectl`, follow [the official documentation](https://kubernetes.io/docs/tasks/tools/#kubectl).
 
 ## Extracting the kubeconfig file
 
-Click on the cluster to expand its properties, and open *KubeConfig*.
-
-![KubeConfig tab in {{k8s_management_service}} Shoot view](assets/shoot_kubeconfig.png)
-
-Click the blue button labeled *Download KubeConfig*. Almost
-instantaneously, in the default download folder of your computer, you
-will get a configuration file with a name similar to
-`kubeconfig--<cluster_name>--<region_name>--<project_id>.yaml`. Create a
-directory named `.kube` in your local user's home, then move the YAML
-file you downloaded into it. Rename the YAML to `config`, ending up with
-`~/.kube/config`.
-
-> By default, `kubectl` searches for its configuration in
-> `~/.kube/config`, but you can [modify this
-> behavior](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
-> by setting the `KUBECONFIG` environment variable, if needed. For
-> example, if you want to retain the original filename as downloaded,
-> you might use:
->
-> ```bash
-> export KUBECONFIG=~/.kube/kubeconfig--<cluster_name>--<region_name>--<project_id>.yaml
-> ```
->
-> You may prefer this approach if you manage multiple Kubernetes
-> clusters.
+To access your shoot cluster, choose between a *static* and a *certificate-based* kubeconfig file.
```

Comment:
Come the availability of Kubernetes 1.27, the static option will no longer be there. So we should not sound here like it's something that the customer can generally choose.

Reply:
Hmmm this was marked resolved but there was no change — was that intentional?

Reply:
Well, I made specific changes in the paragraphs *after* the sentence above, forgot to modify the sentence itself, and accidentally clicked the "Resolve" button. All should be good now.